examples/models/llama2/export_llama_lib.py

```diff
@@ -293,6 +293,12 @@ def build_args_parser() -> argparse.ArgumentParser:
         action="store_true",
         help="This option is only for coreml, and is only supported for MacOS15+/iOS18+",
     )
+    parser.add_argument(
+        "--coreml-quantize",
+        default=None,
+        choices=["b4w"],
+        help="This option is only for coreml: Use coreml quantization",
```

Comment:
Looks like it's block wise 4 bit weight quant...maybe add more description in the help?

Reply:
Something like
```
help="This option is only for coreml: Use coreml quantization, e.g. b4w (blockwise 4-bit weight)",
```
?